### PR TITLE
[14.0][MIG] helpdesk _timesheet_fieldservice

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

File: setup/osi_analytic_segments_defaults/odoo/addons/osi_analytic_segments_defaults [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../osi_analytic_segments_defaults
```
Comment (patrickrwilson) [inline setup/osi_analytic_segments_defaults/odoo/addons/osi_analytic_segments_defaults]: @Freni-OSI It looks like there are segment item changes in this PR, can you clean that up then this should be able to be reviewed.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
It looks like there are segment item changes in this PR, can you clean that up then this should be able to be reviewed.

File: setup/osi_analytic_segments_defaults/odoo/addons/osi_analytic_segments_defaults (hunk 1 above)
Comment (Freni-OSI) [reply setup/osi_analytic_segments_defaults/odoo/addons/osi_analytic_segments_defaults]: Done. Thanks

**Review by patrickrwilson — APPROVED** [review]
Tested and works, LGTM

**Review by osi-scampbell — APPROVED** [review]

**Pull request merged**